### ISSUE TITLE
Adding background image to first page of text document

### ISSUE THREAD

**Issue by j-dimension** [tdf/odftoolkit#186]
Hi, 
I am trying to accomplish the following:

- add an image to 
- the first page of a document
- while keeping it in background, with text being allowed to freely "flow" over the image

I am struggling to make this happen using the API. Could you provide some hints on where to start?

Thank you so much.

**Comment by svanteschubert** [comment]
Hello Jens,

the ODF feature you desire is called in LibreOffice "Title Page" using the "First Page" page style.
The background image is not added as paragraph content but is part of the page style the paragraph style is pointing to.

I did a grep for "First_20_Page" in the ODFDOM sources and it seems this feature is not supported yet, but I would recommend creating an ODT as a template with LibreOffice with an enhanced look & feel anyway and afterwards alter/fill it via ODFDOM API.

I will add an [example ODT called FirstPage.odt](https://github.com/tdf/odftoolkit/files/10002378/FirstPage.odt)
to the issue and would like to point to LibreOffice functionality:

1. Menu->Format->Title Page: https://help.libreoffice.org/7.3/en-GB/text/swriter/01/title_page.html
To choose some pages as Title Pages (I did this by manually editing the page styles in the content.xml)
2. Menu->Format-> Page Style -> Area: https://help.libreoffice.org/7.3/en-GB/text/shared/01/05210100.html
Here I have added a background image from Almeria (some prior LibreOffice Conference location).

Hope this helps!
Svante
